CONTRIBUTING.rst

```diff
@@ -103,22 +82,59 @@ To build the documentation, using  `Sphinx <http://www.sphinx-doc.org/>`_, run:
     environment variable ``DIRECTIVE_SNAPSHOT_QT`` set to ``True``.
 
 
-How-to run the tests
---------------------
+Run the tests
+-------------
+
+- `Install silx for development`_.
+- From the silx project root folder, use `pytest <https://docs.pytest.org/en/stable/how-to/usage.html>`_:
+
+  * To run the complete test suite::
```

Comment:
I think a warning is due here since the flashing windows of the tests suite can be very jarring. 

```suggestion
.. warning::
     
     GUI tests are part of the complete test suite and will make windows appear and disappear very quickly.
     
      **Do not run these if you have a history of epilepsy or motion sickness** 
      
  * To run the complete test suite::
```

Reply:
With xvfb installed on the computer you don't get it, but otherwise, yes it is crazy